README.md

```diff
@@ -36,6 +36,7 @@ The package can be installed as:
     username: "your.name@your.domain", # or {:system, "SMTP_USERNAME"}
     password: "pa55word", # or {:system, "SMTP_PASSWORD"}
     tls: :if_available, # can be `:always` or `:never`
+    tls_versions: [:"tlsv1", :"tlsv1.1", :"tlsv1.2"], # additional details: http://erlang.org/doc/man/ssl.html
```

Comment:
At first I didn't understand why we needed several `tls_versions`.
In order to make it more clear, I would suggest to rename `tls_versions` to something like `supported_tls_versions`. It might be easier to understand we expect the list of all TLS versions supported by the server... I don't know...

What's your opinion?